sentry-android-timber/build.gradle.kts

```diff
@@ -34,6 +33,7 @@ android {
 
     kotlinOptions {
         jvmTarget = JavaVersion.VERSION_1_8.toString()
+        kotlinOptions.languageVersion = Config.kotlinCompatibleLanguageVersion
```

Comment:
thats a breaking change from 1.2 to 1.4, Kotlin 1.2 isn't supported on Kotlin 1.5 anymore

Reply:
Btw, why do we stick to the older version of TImber? Java compat.? They say in 5.0.0 that it should be binary compatible with 4.x

Reply:
that's about Kotlin compatibility and not Timber compatibility, we could technically upgrade Timber to v5 if that's the case, it'd still be a breaking change for users relying on Kotlin 1.2 compatibility.